support-models/src/main/scala/com/gu/support/encoding/DiscriminatedType.scala

```diff
@@ -6,7 +6,7 @@ import io.circe.generic.decoding.DerivedDecoder
 import io.circe.generic.encoding.DerivedAsObjectEncoder
 import io.circe.generic.semiauto.{deriveDecoder, deriveEncoder}
 import io.circe.syntax._
-import io.circe._
+import io.circe.{Codec => _, _}
```

Comment:
this is odd!

Reply:
very odd! we have a "Codec" in the same package as this file.
Previously in scala 2.12 it picked that one.  Since scala 2.13 it now seems to prioritise the one in the import.
So I had to redirect the imported one to `_` so that only the required one is available.